### PR TITLE
[RM][RC1] Move content from CH08 to RC

### LINKED ISSUE #1331: [RM Ch08] Move content of CH08 to RC
As discussed in TSM meeting, many/all content exist in RM Ch08 already covered by RC. @kedmison are you happy with the approach of moving the content away to RC-1?

### PR DESCRIPTION
**Pull request by collivier**
It's duplicated and slightly updated by RC Assumptions & Dependencies
It formats RC Assumptions & Dependencies and removes html when
possible.

Closes #1331

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Comment by markshostak** [comment]
@ All, This topic is on the agenda for review in today's (25/3) RM mtg.

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 1]
```diff
@@ -191,17 +197,23 @@ The below set of steps define the compliance, verification and Conformance proce
...
   * VNF Developer/Supplier validations to be performed documented and supplied
   * NFVI validations to be performed supplied (e.g. echo, manifest)
   * Test to ensure users are added and have correct privileges for the tenant
   * Test to ensure quota against submitted request for the respective tenant
-  * Test to ensure custom flavours against submitted request for respective tenants
+  * Test to ensure custom flavours against submitted request for respective
+    tenants
 
-**Exit criteria**: NFVI Conformance testing should completed with following exit criteria:
+**Exit criteria**: NFVI Conformance testing should completed with following
```
Comment (pgoyal01) [inline doc/ref_cert/lfn/chapters/chapter02.md]: typo: "completed" ---> "complete"

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 2]
```diff
@@ -191,17 +197,23 @@ The below set of steps define the compliance, verification and Conformance proce
...
+exit criteria:
 * All mandatory test cases should pass.
-* Test results collated, centralized, and normalized, with a final report generated showing status of the test scenario/case (e.g. Pass, Fail, Skip, Measurement Success/Fail, etc), along with traceability to a functional, or non-functional, requirement.
+* Test results collated, centralized, and normalized, with a final report
```
Comment (pgoyal01) [inline doc/ref_cert/lfn/chapters/chapter02.md]: British English spelling: centralised normalised

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 3]
```diff
@@ -508,8 +520,8 @@ The validation can then be based on simple pass/fail test results or on a grade
 
 Two categories of metrics are considered depending on whether they are related to either the VNF domain or the NFVI domain itself:
 
-- Metrics related to the VNF domain are defined from VNF perspective (i.e., per VNFC, per vNIC, per vCPU...) and should concern VNF as well as NFVI actors. 
-- Metrics related to the NFVI domain are defined per NFVI node ; their measurement is based on virtual workloads (i.e., VM or container) in order to reflect the performance of a NFVI node with a given profile ; they should only concern NFVI actors. 
+- Metrics related to the VNF domain are defined from VNF perspective (i.e., per VNFC, per vNIC, per vCPU...) and should concern VNF as well as NFVI actors.
```
Comment (pgoyal01) [inline doc/ref_cert/lfn/chapters/chapter02.md]: What does this sentence state and is it necessary?

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 4]
```diff
@@ -508,8 +520,8 @@ The validation can then be based on simple pass/fail test results or on a grade
...
+- Metrics related to the NFVI domain are defined per NFVI node ; their measurement is based on virtual workloads (i.e., VM or container) in order to reflect the performance of a NFVI node with a given profile ; they should only concern NFVI actors.
```
Comment (pgoyal01) [inline doc/ref_cert/lfn/chapters/chapter02.md]: can you clarify, ".. NFVI node with a given profile"

typo extra space before ";"

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 5]
```diff
@@ -191,17 +197,23 @@ The below set of steps define the compliance, verification and Conformance proce
   * Images uploaded to tenant space
   * External system test data needs identified
   * Owners (NFVI, VNF, PTL, etc) documented
-  * Security Compliance Satisfied (Refer to CNTT Chapter XXXX Security for additional tests, scans, and vulnerabilities validations)
+  * Security Compliance Satisfied (Refer to CNTT Chapter XXXX Security for
```
Comment (pgoyal01) [inline doc/ref_cert/lfn/chapters/chapter02.md]: should this state, "(Refer to [RM Chapter 07](../../ref_model/chapters/chapter07.md) for additional security requirements)"  --

**Comment by collivier** [comment]
@pgoyal01 Thank you for your reviews. My proposal would be simply to move all contents from RM Chapter8 to RC (these 2 commit are about full duplicated sections) as expected by TSC. I would consider fixing the typos is quite out of scope (they are in both sides) and could have been fixed in a second PR. If it helps I could apply your fixes in a new commit here but if it results in big changes, it should be managed by an RM contributor which I'm not ;).

**Comment by collivier** [comment]
> @ All, This topic is on the agenda for review in today's (25/3) RM mtg.

@markshostak Let's me know if I continue here or if RM contributor want to take the lead here. Be free to do so.

**Comment by fuqiao123** [comment]
My understanding is this is a copy-paste effort from RM chp 8 to RC, am I right @collivier ? If so, I think we can merge this first, and if there are any mis-match between RM and RC, we can have RM team submit another PR.

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 6]
```diff
@@ -171,17 +171,23 @@ The below set of steps define the compliance, verification and Conformance proce
...
+  * Lab assets/resources and respective software revision levels are specified,
+    with confirmation of compatibility across external systems
   * Tenant needs identified
-  * All connectivity, network, image, VMs, delivered with successful pairwise tests
+  * All connectivity, network, image, VMs, delivered with successful pairwise
+    tests
   * Lab instrumented for proper monitoring
-  * Lab needs to be setup according to RA1/RA2 as defined by CNTT and should be in the NFVI required state.
+  * Lab needs to be setup according to RA1/RA2 as defined by CNTT and should be
+    in the NFVI required state.
```
[the remark this answers is not in the document]
Comment (collivier) [reply doc/ref_cert/lfn/chapters/chapter02.md:190]: Only a few words were added in RC (git diff). By default, we do consider RC as a copy/paste + a very small update here.

File: doc/ref_cert/lfn/chapters/chapter02.md [hunk 7]
```diff
@@ -191,17 +197,23 @@ The below set of steps define the compliance, verification and Conformance proce
...
 
-**Exit criteria**: NFVI Conformance testing should completed with following exit criteria:
+**Exit criteria**: NFVI Conformance testing should completed with following
+exit criteria:
 * All mandatory test cases should pass.
-* Test results collated, centralized, and normalized, with a final report generated showing status of the test scenario/case (e.g. Pass, Fail, Skip, Measurement Success/Fail, etc), along with traceability to a functional, or non-functional, requirement.
+* Test results collated, centralized, and normalized, with a final report
+  generated showing status of the test scenario/case (e.g. Pass, Fail, Skip,
+  Measurement Success/Fail, etc), along with traceability to a functional, or
+  non-functional, requirement.
```
[the remark this answers is not in the document]
Comment (collivier) [reply doc/ref_cert/lfn/chapters/chapter02.md:216]: Yes. It's formatted as the whole CNTT doc should be.

**Comment by markshostak** [comment]
@collivier Bon jour Cedric, We've reviewed and discussed the content in RMCH-8. In a nutshell, please remove it all from RM. However, there are some specific sections to pay attention to, as discussed in the last (1/4/2020) RM call:
- 8.2.1 {1 of 2} (RI Requirements): Please be sure this moves to RC-1
- 8.2.4 (Requirements): Please be sure this moves to RC-1
- 8.3.1 (Entry/Exit Criteria): Please be sure this moves to RC-1

We believe the content above is important and not already present in RC1.
All other sections of RMCH-8 can be moved if not already present in RC1. Any content that's duplicated can simply be purged from RMCH-8.

If you have any questions about moving the above sections, please address to @rabi-abdel.
If you have any questions about any remaining content, please address to myself and/or @kedmison.

Merci Beaucoup!
-Mark

**Comment by collivier** [comment]
Yes. This two commits suggested here were double checked via git diff.
As far as I remember only one half sentence was added in RC.
I would suggest to assign this pull/request to a RM contributor or an overall editor It doesn't make so much I continue for the last part.

Be free to continue this PR or open a new one.

**Comment by fuqiao123** [comment]
According to discusson on Apr 8 RC WS call, we will merge this PR first. And create a new one to cover the review

**Pull request merged**